editor/block-mover/test/mover-label.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { getBlockMoverLabel, getMultiBlockMoverLabel } from '../mover-label';
+
+describe( 'block mover', () => {
+	const dir_up = -1,
```

Comment:
Just to clarify - the variables `dirUp` and `dirDown` exist purely to guide future testers as to what the last argument is for. One could argue "why not just have all the arguments named", but the others felt too varied to be worth being named. Am happy to just remove the vars though, if you feel it would be more consistent with the rest of the test suites.